src/web/pages.rkt

```diff
@@ -39,10 +26,11 @@
   (define ctx (test-context test))
   (match page
     ["graph.html"
-     (match result
-       [(? test-success?) (make-graph result out (get-interactive-js result ctx) profile?)]
-       [(? test-timeout?) (make-traceback result out profile?)]
-       [(? test-failure?) (make-traceback result out profile?)])]
+     (match (test-result-status result)
+       ['success (make-graph result out (get-interactive-js result ctx) profile?)]
+       ['timeout (make-graph result out (get-interactive-js result ctx) profile?)]
+       ['failure (make-traceback result out profile?)]
```

Comment:
I think this doesn't match what happened before? Also shouldn't there be a `make-timeout` page? Very confused.

Reply:
Oops, copied incorrectly.